<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QLabel
{
	border: 2px solid gray;
	padding: 0 8px;
	background: white;
}

QPushButton
{
background: white;
border: 2px solid black;
border-radius: 2px;
}

#btny1
{
background-color: rgb(252, 2, 128);
}


#btnx1
{
background-color: magenta;
}

#btny2
{
background-color: lightblue;
}


#btnx2
{
background-color: blue;
}

#btnXAxis
{
background-color: yellow;
}



#btnYAxis
{
background-color: cyan;
}

#btnRotary
{
background-color: gray;
}



</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rdbCam">
        <property name="text">
         <string>Camera</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">background: none;
border: black;</string>
          </property>
          <property name="text">
           <string>Angle:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblTheta">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnRotary">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Rotary</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">border: none;
background: none;</string>
          </property>
          <property name="text">
           <string>Position A_T 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="at1lbl">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnx1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>x1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btny1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>y1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">border: none;
background: none;</string>
          </property>
          <property name="text">
           <string>Position A_T 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="at2lbl">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnx2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>x2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btny2">
            <property name="text">
             <string>y2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">background: none;
border: black;</string>
          </property>
          <property name="text">
           <string>Main Position:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblOrigin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnXAxis">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>X-Axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnYAxis">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Y-Axis</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">background: none;
border: none;
</string>
          </property>
          <property name="text">
           <string>Input Voltage:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblVolts">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">background: none;
border: none;
</string>
          </property>
          <property name="text">
           <string>Last Command</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblCmdOut">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="btnSetA_tSpeed">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.483, x2:0, y2:0.505955, stop:0 rgba(56, 0, 255, 255), stop:1 rgba(255, 0, 208, 255))</string>
          </property>
          <property name="text">
           <string>Set A_T Speeds</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSetMainSpeed">
          <property name="styleSheet">
           <string notr="true">background-color:qlineargradient(spread:pad, x1:1, y1:0.506, x2:0, y2:0.505955, stop:0 rgba(0, 246, 255, 255), stop:1 rgba(255, 255, 0, 255))</string>
          </property>
          <property name="text">
           <string>Set Main Speeds</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnKill">
          <property name="text">
           <string>Kill</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnQuit">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnHome">
          <property name="text">
           <string>Go Home</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnAdvanced">
          <property name="text">
           <string>Advanced
User Mode</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="btnStoreCmd">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Store Cmnd</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnReadCmd">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Read Cmnd</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSendCmd">
          <property name="text">
           <string>Send Cmnd</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lblCustomCmd">
            <property name="styleSheet">
             <string notr="true">
border: none;
background:none;</string>
            </property>
            <property name="text">
             <string>Custom Command:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lblCmdCheckDesc">
            <property name="styleSheet">
             <string notr="true">background: none;
border: none;</string>
            </property>
            <property name="text">
             <string>Current Comand:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblCmdCheck">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnAverageUser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>1666666</width>
            <height>166666</height>
           </size>
          </property>
          <property name="text">
           <string>Standard
User Mode</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="Camerabtnwid" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnTiltUp">
         <property name="text">
          <string>Tilt Up</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="btnPanRight">
           <property name="text">
            <string>Pan Right</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPanLeft">
           <property name="text">
            <string>Pan Left</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnTiltDown">
         <property name="text">
          <string>Tilt Down</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionWin">
   <property name="text">
    <string>Win</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btnQuit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
